test/test_runs.py

```diff
@@ -217,47 +303,63 @@ def test_run_05_import(capsys: Any):
     assert os.path.exists(xcfn)
 
     # Run import.
-    sys.argv = [fct, '-v', '-n', NAME, '-g', str(GPORT), '-f', NAMEZP, '-x', xcfn]
+    sys.argv = [fct, '-v', '-n', name, '-g', str(gport), '-f', namezp, '-x', xcfn]
     ximport.main()
     out, err = capsys.readouterr()
     print(f'out=<<<{out}>>>')
     print(f'err=<<<{err}>>>')
-    assert NAMEPG in err
-    assert f'http://localhost:{GPORT}' in err
+    assert namepg in err
+    assert f'http://localhost:{gport}' in err
     os.unlink(xcfn)
-    sys.argv = sargs
 
 
-def test_run_06_export(capsys: Any):
-    'test export'
-
-    sargs = sys.argv
+@insulate_globals
+@pytest.mark.parametrize(
+    'name,gport,name2,gport2',
+    [
+        (NAME, GPORT, NAME2, GPORT2),
+    ],
+)
+def test_run_07_export(capsys: Any, name: str, gport: int, name2: str, gport2: int):
+    '''Export data into the test project.
+
+    Args:
+        capsys: Pytest fixture for capturing stdout/stderr.
+        name: The grape project name for this test.
+        gport: The grafana port for this test.
+        name2: The export grape project.
+        gport2: The export grafana port.
+    '''
+    namegr, namepg, namezp = make_names(name)
     fct = inspect.stack()[0].function
 
-    # Create the NAME2 container to export to.
-    sys.argv = [fct, '-v', '-n', NAME2, '-g', str(GPORT2)]
+    # Prerequisites.
+    assert os.path.exists(namepg)
+
+    # Create the name2 container to export to.
+    sys.argv = [fct, '-v', '-n', name2, '-g', str(gport2)]
     create.main()
     out, err = capsys.readouterr()
     print(f'out=<<<{out}>>>')
     print(f'err=<<<{err}>>>')
     assert os.path.exists(NAMEPG2)
-    assert NAMEGR2 in err
-    assert NAMEPG2 in err
+    assert namegr2 in err
+    assert namepg2 in err
```

Comment:
Where's `namepg2` defined? Shouldn't there be a call to `make_names` for 1 and 2?

Reply:
Great catch! But how the heck could this pass pylint?!?

Reply:
It passed pylint because the decorator was confusing pylint and pytest.